docs/Configmap_Properties.md

```diff
@@ -11,7 +11,7 @@ This document explains configmap variables.
 | webhook.enabled                   |  Enable webhook?              | false              | Required           |
 | iam.role.max.limit.per.namespace      | Maximum number of roles per namespace |   1        | Required |
 | aws.region                        | AWS Region                    | us-west-2          | Required |
-| iam.default.trust.policy.role.arn.list| Default trust policy role arn list |           | Optional |
+| iam.default.trust.policy| Default trust policy role. This must follow v1alpha1.AssumeRolePolicyDocument syntax|           | Optional |
```

Comment:
Initially, with trust.policy.role.arn.list, user could provide list of trust policies. But now we restrict it to one? As, iam.default.trust.policy will take one json string as input.

Reply:
If you look at the sample from above, Statement field is an array and user can include as many as possible and that too we no longer restricting to only "AWS". This means, user can use sts:AssumRole or sts:AssumeRoleWithWebIdentity with multiple combinations.
```
  iam.default.trust.policy: '{"Version": "2012-10-17","Statement": [{"Effect": "Allow","Principal":
    {"AWS": "arn:aws:iam::{{.AccountID}}:role/iks-kjain3-20200504-k8s-kiam-role"},"Action":"sts:AssumeRole"},{"Effect":"Allow","Action":"sts:AssumeRoleWithWebIdentity","Principal":{"Federated":"arn:aws:iam::{{.AccountID}}:oidc-provider/oidc.eks.us-west-2.amazonaws.com/id/8BC1FC2A37677AEB320AB138273EF638"},"Condition":{"StringLike":{"oidc.eks.us-west-2.amazonaws.com/id/8BC1FC2A37677AEB320AB138273EF638:sub":"system:serviceaccount:{{.NamespaceName}}:*"}}}]}'
```